torch_xla/csrc/tensor_util.cpp

```diff
@@ -792,6 +714,64 @@ at::Tensor XlaLiteralToTensorHelper(const xla::Literal& literal,
 
 }  // namespace
 
+void PopulateTensorBuffer(const at::Tensor& tensor,
```

Comment:
hmm, any reason to move this function down?

Reply:
I had to move it out of the anonymous namespace to expose it outside of the file